spec/integration/admin_api/big_json.json

```diff
@@ -0,0 +1 @@
19,192.231.66.0/24,192.234.189.0/24,192.245.101.0/24,193.0.129.0/24,193.23.126.0/24,193.25.48.0/20,193.26.64.0/19,193.43.134.0/24,193.104.41.0/24,193.104.94.0/24,193.104.110.0/24,193.105.207.0/24,193.105.245.0/24,193.107.16.0/22,193.138.244.0/22,193.139.0.0/16,193.150.120.0/24,193.164.11.0/24,193.177.64.0/18,193.189.116.0/23,193.222.50.0/24,193.243.0.0/17,194.0.177.0/24,194.1.152.0/24,194.29.185.0/24,194.38.0.0/18,194.50.116.0/24,194.54.156.0/22,194.110.160.0/22,195.20.141.0/24,195.68.222.0/23,195.78.108.0/23,195.88.190.0/23,195.182.57.0/24,195.190.13.0/24,195.191.56.0/23,195.191.102.0/23,195.225.176.0/22,195.238.180.0/22,196.1.109.0/24,196.42.128.0/17,196.63.0.0/16,196.188.0.0/14,196.193.0.0/16,196.247.0.0/16,197.154.0.0/16,198.13.0.0/20,198.14.128.0/19,198.14.160.0/19,198.20.16.0/20,198.23.32.0/20,198.45.32.0/20,198.45.64.0/19,198.48.16.0/20,198.56.64.0/18,198.57.64.0/20,198.62.70.0/24,198.62.76.0/24,198.96.224.0/20,198.99.117.0/24,198.102.222.0/24,198.148.212.0/24,198.151.16.0/20,198.151.64.0/18,198.151.152.0/22,198.160.205.0/24,198.162.208.0/20,198.169.201.0/24,198.177.175.0/24,198.177.176.0/22,198.177.180.0/24,198.177.214.0/24,198.178.64.0/19,198.179.22.0/24,198.181.32.0/20,198.181.64.0/19,198.183.32.0/19,198.184.193.0/24,198.184.208.0/24,198.186.25.0/24,198.186.208.0/24,198.187.64.0/18,198.187.192.0/24,198.190.173.0/24,198.199.212.0/24,198.202.237.0/24,198.204.0.0/21,198.205.64.0/19,198.212.132.0/24,199.5.152.0/23,199.5.229.0/24,199.26.96.0/19,199.26.137.0/24,199.26.207.0/24,199.26.251.0/24,199.33.145.0/24,199.33.222.0/24,199.34.128.0/18,199.46.32.0/19,199.58.248.0/21,199.60.102.0/24,199.71.56.0/21,199.71.192.0/20,199.84.55.0/24,199.84.56.0/22,199.84.60.0/24,199.84.64.0/19,199.87.208.0/21,199.88.32.0/20,199.88.48.0/22,199.89.16.0/20,199.89.198.0/24,199.120.163.0/24,199.165.32.0/19,199.166.200.0/22,199.184.82.0/24,199.185.192.0/20,199.196.192.0/19,199.198.160.0/20,199.198.176.0/21,199.198.184.0/23,199.198.188.0/22,199.200.64.0/19,199.212.96.0/20,199.223.0.0/20,199.230.64.0/19,199.230.96.0/21,199.233.85.0/24,199.233.96.0/24,199.245.138.0/24,199.246.137.0/24,199.246.213.0/24,199.246.215.0/24,199.248.64.0/18,199.249.64.0/19,199.253.32.0/20,199.253.48.0/21,199.253.224.0/20,199.254.32.0/20,200.3.128.0/20,200.22.0.0/16,201.169.0.0/16,201.182.0.0/16,202.0.192.0/18,202.20.32.0/19,202.21.64.0/19,202.27.96.0/23,202.27.98.0/24,202.27.99.0/24,202.27.100.0/22,202.27.120.0/22,202.27.161.0/24,202.27.162.0/23,202.27.164.0/22,202.27.168.0/24,202.39.112.0/20,202.40.32.0/19,202.40.64.0/18,202.61.108.0/24,202.68.0.0/18,202.80.152.0/21,202.148.32.0/20,202.148.176.0/20,202.183.0.0/19,202.189.80.0/20,203.0.116.0/22,203.2.200.0/22,203.9.0.0/19,203.31.88.0/23,203.34.70.0/23,203.34.71.0/24,203.86.252.0/22,203.148.80.0/22,203.149.92.0/22,203.169.0.0/22,203.189.112.0/22,203.191.64.0/18,204.19.38.0/23,204.44.32.0/20,204.44.192.0/20,204.44.224.0/20,204.48.16.0/20,204.52.255.0/24,204.57.16.0/20,204.75.147.0/24,204.75.228.0/24,204.80.198.0/24,204.86.16.0/20,204.87.199.0/24
```

Comment:
Instead of having to deal with such a fixture, simply use `string.rep(bytes)` to build a fixture body.